ui/src/app/utils/authentication.tsx

```diff
@@ -70,7 +75,7 @@ export function useAuthentication() {
   const {config} = useStore(serverConfigStore);
 
   useEffect(() => {
-    if (config && (!environment.allowTestAccessTokenOverride || !window.localStorage.getItem(LOCAL_STORAGE_KEY_TEST_ACCESS_TOKEN))) {
+    if (config && !useAccessToken) {
```

Comment:
careful, this always evaluates to false now. you need to actually invoke the function: `!useAccessToken()`